crates/core/src/kernel/expressions/scalars.rs

```diff
@@ -433,6 +445,16 @@ impl PrimitiveType {
                     .ok_or(self.parse_error(raw))?;
                 Ok(Scalar::Timestamp(micros))
             }
+            TimestampNtz => {
+                let timestamp = NaiveDateTime::parse_from_str(raw, "%Y-%m-%d %H:%M:%S%.f")
```

Comment:
So the rough part of this is that the format for the timestamp can be either `1970-01-01 00:00:00`, or `1970-01-01 00:00:00.123456`, I don't believe this format would cover both right?

Reply:
Not sure, I saw the mention in the protocol on both formats. But timestamp was only parsed with the ms precision. Maybe @roeap can chip in here?

Reply:
https://github.com/delta-io/delta/blob/master/PROTOCOL.md#timestamp-without-timezone-timestampNtz it talks about supporting both formats for this datatype here

Reply:
I checked, it has no problem parsing both formats with "%Y-%m-%d %H:%M:%S%.f", both can be parsed into a timestamp